### PR TITLE
nice.md: Including common nice commands.

### PR DESCRIPTION
**Pull request by shadowfax92**

### CODE REVIEW

**Comment by waldyrious** [comment]
Instead of "altered", I'd say "custom" would be more self-explanatory.

**Comment by shadowfax92** [comment]
I think "altered" makes more sense. man pages too says "altered"

**Comment by waldyrious** [comment]
Fair enough. I stand by my suggestion, so let's see what others have to say. In any case, thanks for the contribution!

**Comment by igorshubovych** [comment]
Have you tried to render this page with
`tldr --render path/to/your/page/file.md`

I am asking because descriptions looks too long for me.

**Comment by igorshubovych** [comment]
I would rephrase it.

~~~Markdown

> Execute a program with an custom scheduling priority (niceness).
> Niceness values range from -20 (the highest priority) to 19 (the lowest).
~~~

**Comment by shadowfax92** [comment]
@igorshubovych it was getting rendered. Including suggestions.

**Comment by ostera** [comment]
LGTM

**Comment by felixonmars** [comment]
:+1:

**Comment by waldyrious** [comment]
Since this requires a rebase, I'd suggest taking the opportunity to fix also "an custom" --> "a custom".

**Comment by shadowfax92** [comment]
@waldyrious done.

**Comment by waldyrious** [comment]
Alright! Let's get this merged :)
